src/Core/Config/ConfigCacheLoader.php

```diff
@@ -0,0 +1,183 @@
+<?php
+
+namespace Friendica\Core\Config;
+
+/**
+ * The ConfigCacheLoader loads config-files and stores them in a ConfigCache ( @see ConfigCache )
+ *
+ * It is capable of loading the following config files:
+ * - *.config.php   (current)
+ * - *.ini.php      (deprecated)
+ * - *.htconfig.php (deprecated)
+ */
+class ConfigCacheLoader
+{
+	/**
+	 * The Sub directory of the config-files
+	 * @var string
+	 */
+	const SUBDIRECTORY   = '/config/';
+	/**
+	 * The addon sub-directory
+	 * @var string
+	 */
+	const ADDONSDIRECTORY = '/addons/';
```

Comment:
The addon folder as defined in `src/Core/Addon.php` is `/addon/`, please make it a constant in the `Addon` class instead with the correct value.